### PR TITLE
Laurent/lp 48 update rebranding strings translations

### PR DESCRIPTION
**Pull request by f5soh**
Update strings and keep generic when possible (wizard...)

Updated french translations

### CODE REVIEW

**Comment by parched** [comment]
Look great, the only thing you could change is use a GCS define (which by default contains "LibrePilot GCS") in a few of those places.

Another thing that we should do is make those defines in openpilotgcs.pri so every file can use them.

File: ground/openpilotgcs/src/plugins/coreplugin/generalsettings.cpp [hunk 1]
```diff
@@ -233,7 +233,7 @@ void GeneralSettings::setLanguage(const QString &locale)
     if (m_language != locale) {
         if (!locale.isEmpty()) {
             QMessageBox::information((QWidget *)Core::ICore::instance()->mainWindow(), tr("Restart required"),
-                                     tr("The language change will take effect after a restart of the OpenPilot GCS."));
+                                     tr("The language change will take effect after a restart of the %1 GCS.").arg(ORG_BIG_NAME));
```
Comment (parched) [inline ground/openpilotgcs/src/plugins/coreplugin/generalsettings.cpp]: Here you can use GCS_BIG_NAME

File: ground/openpilotgcs/src/plugins/coreplugin/mainwindow.cpp [hunk 2]
```diff
@@ -134,7 +134,7 @@ MainWindow::MainWindow() :
 #endif
     m_toggleFullScreenAction(0)
 {
-    setWindowTitle(tr("LibrePilot GCS ") + VersionInfo::label());
+    setWindowTitle(tr("%1 GCS ").arg(ORG_BIG_NAME) + VersionInfo::label());
```
Comment (parched) [inline ground/openpilotgcs/src/plugins/coreplugin/mainwindow.cpp]: And here

File: ground/openpilotgcs/src/plugins/usagetracker/usagetrackerplugin.cpp [hunk 3]
```diff
@@ -81,11 +81,11 @@ void UsageTrackerPlugin::onAutopilotConnect()
             message.setIcon(QMessageBox::Information);
             message.addButton(tr("Yes, count me in"), QMessageBox::AcceptRole);
             message.addButton(tr("No, I will not help"), QMessageBox::RejectRole);
-            message.setText(tr("Openpilot GCS has a function to collect limited anonymous information about "
-                               "the usage of the application itself and the OpenPilot hardware connected to it.<p>"
+            message.setText(tr("%1 GCS has a function to collect limited anonymous information about "
```
Comment (parched) [inline ground/openpilotgcs/src/plugins/usagetracker/usagetrackerplugin.cpp]: And here

Comment (f5soh) [inline ground/openpilotgcs/src/plugins/usagetracker/usagetrackerplugin.cpp]: I use the same arg for multiples LibrePilot in text (LibrePilot GCS, Librepilot website...)

File: ground/openpilotgcs/src/plugins/coreplugin/generalsettings.cpp (hunk 1 above)
Comment (f5soh) [inline ground/openpilotgcs/src/plugins/coreplugin/generalsettings.cpp]: Not sure is a good idea use GCS_BIG_NAME, in some languages it can be inverted.

Comment (parched) [inline ground/openpilotgcs/src/plugins/coreplugin/generalsettings.cpp]: I guess it depends which way you look at it. I would say "LibrePilot GCS" is a whole product name that isn't part of a language, (it just happens to be based on some french and english words"

File: ground/openpilotgcs/src/plugins/usagetracker/usagetrackerplugin.cpp (hunk 3 above)
Comment (parched) [inline ground/openpilotgcs/src/plugins/usagetracker/usagetrackerplugin.cpp]: I think you can use both. (.arg() twice) Imagine we had 2 gcs, say "LibrePilot GCS" and "LibrePilot Copter Control GCS" that both used this code.

**Pull request merged**